sources/models/src/modeled_types/kubernetes.rs

```diff
@@ -223,12 +223,12 @@ pub struct KubernetesTaintValue {
 lazy_static! {
     pub(crate) static ref KUBERNETES_TAINT_VALUE: Regex = Regex::new(
         r"(?x)^
-       [[:alnum:]]  # at least one alphanumeric
+       ([[:alnum:]]  # values have to start with alphanumeric if they're specified
```

Comment:
nit: I think the existing pattern of formatting is clearer, i.e. grouping on its own line, because it's quicker to see which things are tied together and which are required, and you can document each grouping clearly.  Something like:
```
(
   [[:alnum:]]  # values have to start with alphanumeric if they're specified
   (
      ([[:alnum:]._-]{0,61})?  # characters allowed in middle
      [[:alnum:]]  # values have to end with alphanumeric
   )?  # only the first alphanumeric is required, further chars optional
)?  # the whole taint value is optional
:
```